src/nl/hannahsten/texifyidea/settings/conventions/TexifyConventionsSettingsManager.kt

```diff
@@ -0,0 +1,90 @@
+package nl.hannahsten.texifyidea.settings.conventions
+
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+
+/**
+ * Model for the Texify convention settings.
+ *
+ * A model consists of a project state and a global state and presents a unified view on both. For example, modifying
+ * the currently active scheme modifies either the project state or the global state, depending on which scheme is
+ * selected.
+ */
+data class TexifyConventionsSettings(
```

Comment:
There's many classes related to schemes and settings, many of which are not in different files, which can make it harder to find them.
To have a better overview, can you think of a structure to make it clearer which classes are intended to be called outside this package (mainly to get currently applicable conventions), and which ones are 'package internal' abstractions?

Reply:
I made everything package internal `internal`, reducing the public API to `TexifyConventionsSettings` and `TexifyConventionsSettingsManager` basically.

Reply:
Yes, that should be clear enough